src/handler.rs

```diff
@@ -0,0 +1,234 @@
+use std::path::Path;
+use std::process::{ExitStatus, Stdio};
+use std::time::Duration;
+
+use anyhow::{anyhow, Error, Result};
+use dprint_core::configuration::{
+  resolve_new_line_kind, ConfigKeyMap, GlobalConfiguration, NewLineKind, ResolveConfigurationResult,
+};
+use dprint_core::plugins::{PluginHandler, PluginInfo};
+use handlebars::Handlebars;
+use serde::{Deserialize, Serialize};
+use tokio::io::{AsyncBufReadExt, AsyncRead, AsyncWriteExt, BufReader};
+use tokio::process::Command;
+use tokio::sync::oneshot::{self, Receiver, Sender};
+use tokio::time::error::Elapsed;
+use tokio::time::timeout;
+
+use crate::configuration::Configuration;
+
+pub struct ExecHandler;
+
+impl PluginHandler<Configuration> for ExecHandler {
+  fn resolve_config(
+    &mut self,
+    config: ConfigKeyMap,
+    global_config: &GlobalConfiguration,
+  ) -> ResolveConfigurationResult<Configuration> {
+    Configuration::resolve(config, global_config)
+  }
+
+  fn get_plugin_info(&mut self) -> PluginInfo {
+    let name = env!("CARGO_PKG_NAME").to_string();
+    let version = env!("CARGO_PKG_VERSION").to_string();
+    PluginInfo {
+      name: name.clone(),
+      version: version.clone(),
+      config_key: "exec".to_string(),
+      file_extensions: vec![], // this configured in the plugins' `associations`
+      file_names: vec![],      // this configured in the plugins' `associations`
+      help_url: env!("CARGO_PKG_HOMEPAGE").to_string(),
+      config_schema_url: format!(
+        "https://plugins.dprint.dev/dprint/{}/{}/schema.json",
+        name, version
+      ),
+      update_url: Some(format!(
+        "https://plugins.dprint.dev/dprint/{}/latest.json",
+        name
```

Comment:
These pages do not exist at the moment. Should I leave them for future, or do you want them to be blank until some documentation is added here https://github.com/dprint/dprint/tree/main/website/plugins?

Reply:
It requires at least one release. It should start working once a release is done.